plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -199,12 +198,6 @@ func (l *persistentVolumeLabel) findVolumeLabels(volume *api.PersistentVolume) (
 
 	// Either missing labels or we don't trust the user provided correct values.
 	switch {
-	case volume.Spec.GCEPersistentDisk != nil:
-		labels, err := l.findGCEPDLabels(volume)
-		if err != nil {
-			return nil, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err)
-		}
-		return labels, nil
 	case volume.Spec.AzureDisk != nil:
```

Comment:
Maybe for a separate PR, but the Azure and vSphere provider should be removed as well.

Reply:
Actually, with those providers removed the entire admission plugin can be removed as well

Reply:
From my understanding, there should be very few users still depending on this admission controller. But for rare cases where they still need this behavior, we created an external webhook they can install instead: https://github.com/kubernetes-sigs/cloud-pv-admission-labeler 

cc @msau42 @jsafrane

Reply:
@andrewsykim being removed here - https://github.com/kubernetes/kubernetes/pull/124505

Reply:
once https://github.com/kubernetes/kubernetes/pull/124505 lands, i will remove the entire admission plugin here